modelci/cli/modelhub.py

```diff
@@ -196,3 +197,90 @@ def detail(model_id: str = typer.Argument(..., help='Model ID')):
     with requests.get(f'{app_settings.api_v1_prefix}/model/{model_id}') as r:
         data = r.json()
         model_detailed_view(MLModel.parse_obj(data))
+
+
+@app.command('convert')
+def convert(
+        src: str = typer.Argument(..., help='the prototype model framework to be converted'),
+        dst: str = typer.Argument(..., help='the destination model framework after converting'),
+        model_path: str = typer.Argument(..., help='path of the model to be converted'
+                                                   'such as \'/home/model\' for keras savedmodel or'
+                                                   '\'/home/model/xxx.suffix\' for a file'),
+        model_name: str = typer.Option(..., '-n', '--name', help='model name'),
+        version: Optional[int] = typer.Option(..., '-v', '--version', min=1, help='Version number'),
+        shape: str = typer.Option(None, '-s', '--shape', help='input shape of model, such as\'-shape [28,28,1]\''),
+        datatype: str = typer.Option(None, '-d', '--datatype', help='data type of model input, such as\'float32\'')
```

Comment:
We have an input / output shape auto conversion, check here:
https://github.com/cap-ntu/ML-Model-CI/blob/bf373dbc05c0efea6d92a42b5cbd92dc0e35240e/modelci/cli/modelhub.py#L53-L58

Reply:
When typing "{"name": "input", "shape": [-1, 3, 224, 224], "dtype": "TYPE_FP32", "format": "FORMAT_NCHW"}" in the CLI, The following exception comes out:
           "Expecting property name enclosed in double quotes: line 1 column 2 (char 1) (type=value_error.jsondecode; msg=Expecting property name enclosed in double quotes; doc={name:; pos=1; lineno=1; colno=2)"

Therefore I highly recommend to hide IOShape from users for interface friendly. Since I am not familiar with typer CLI  for witch apparently I made mistakes when typing args for IOShape in CLI, and either are the users.  

It is better to set arguments like 'name'   'shape' ... for users to fill but a complicated list of IOShape

Reply:
This 'IOShape' is different from the one defined in 'bo', and they can not substitute for each other. I am having trouble make IOShape(defined in bo) a parameter of convert CLI, it would be a great help if anyone can help me with it because I am too unfamiliar with Typer. For the time being I still use str as parameters and transform them into IOShape in functions.

Reply:
Please use the version in `types.models.common.py` instead. the `bo` one is deprecated.

Reply:
Also, try testing using `python modelci/cli/__init__.py modelhub xxxx`. The Typer fixture only works at the root CLI level.

Reply:
Do you mean that the `bo` one will be removed sometime later? If so I will add a TODO here. Because `IOShape` in `bo` has been used as standard shape in every convert function and unit test. It is highly recommended to rewrite them with the `types.models.common.py IOShape` before further developing.  Or is it better to first fix the IOShape problem before this PR for convert CLI?